packages/sites/src/update-site.ts

```diff
@@ -15,73 +16,73 @@ import { updateItem, IUpdateItemResponse } from "@esri/arcgis-rest-portal";
  * Update an existing site item
  * This function supports the equivalent of a PATCH REST operation
  * It will fetch the current item from ago, and then apply
- * a subset of property changes to the model if a patchList is included.
- * The patchList can include any property paths on the item.
+ * a subset of property changes to the model if a allowList is included.
+ * The allowList can include any property paths on the item.
  * If the list is empty, then the entire site model is overwritten.
  * @param {Object} model Site Model to update
- * @param {Array} patchList Array of property paths to update
- * @param {IHubRequestOptions} hubRequestOptions
+ * @param {IUpdateSiteOptions} updateSiteOptions
  */
 export function updateSite(
```

Comment:
I don't disagree, but it looks like [update-page](https://github.com/Esri/hub.js/blob/master/packages/sites/src/pages/update-page.ts#L32) already differs from `update-page` in that it can receive 2 or 3 args and there's some variable reassignment to account for this.

Reply:
In an ideal world, we'd keep these signatures pretty similar... but they are different item types, so I don't think we should get too wound up about it

Reply:
Thanks for your input @dbouwman. Will hold off on making this change for now then.

Reply:
FYI - though, this was already on the [list of things to change at the next breaking version](https://github.com/Esri/hub.js/issues/305): 

https://github.com/Esri/hub.js/blob/master/packages/sites/src/pages/update-page.ts#L32-L35

Reply:
Pushed another commit to add `IUpdatePageOptions` and refactor `updatePage` to use per side-convo w/ @tomwayson